components/scream/src/physics/rrtmgp/scream_rrtmgp_interface.cpp

```diff
@@ -1,13 +1,284 @@
 #include "scream_rrtmgp_interface.hpp"
+#include "mo_load_coefficients.h"
+#include "mo_load_cloud_coefficients.h"
+#include "mo_gas_concentrations.h"
+#include "mo_gas_optics_rrtmgp.h"
+#include "mo_cloud_optics.h"
+#include "mo_rte_sw.h"
+#include "mo_rte_lw.h"
+#include "const.h"
 
 namespace scream {
     namespace rrtmgp {
+
+        OpticalProps2str get_cloud_optics_sw(CloudOptics &cloud_optics, GasOpticsRRTMGP &kdist, real2d &p_lay, real2d &t_lay, real2d &lwp, real2d &iwp, real2d &rel, real2d &rei);
+        OpticalProps1scl get_cloud_optics_lw(CloudOptics &cloud_optics, GasOpticsRRTMGP &kdist, real2d &p_lay, real2d &t_lay, real2d &lwp, real2d &iwp, real2d &rel, real2d &rei);
+
+        /*
+         * Names of input files we will need.
+         */
+        std::string coefficients_file_sw = "./data/rrtmgp-data-sw-g224-2018-12-04.nc";
+        std::string coefficients_file_lw = "./data/rrtmgp-data-lw-g256-2018-12-04.nc";
+        std::string cloud_optics_file_sw = "./data/rrtmgp-cloud-optics-coeffs-sw.nc";
+        std::string cloud_optics_file_lw = "./data/rrtmgp-cloud-optics-coeffs-lw.nc";
+
+        /* 
+         * Objects containing k-distribution information need to be initialized
+         * once and then persist throughout the life of the program, so we
+         * declare them here within the rrtmgp namespace.
+         */
+        GasOpticsRRTMGP k_dist_sw;
+        GasOpticsRRTMGP k_dist_lw;
+
+        /*
+         * Objects containing cloud optical property look-up table information.
+         * We want to initialize these once and use throughout the life of the
+         * program, so declare here and read data in during rrtmgp_initialize().
+         */
+        CloudOptics cloud_optics_sw;
+        CloudOptics cloud_optics_lw;
+
+        bool initialized = false;
+
         /* 
          * Define some dummy routines so we can start working on the interface
```

Comment:
Are these still dummy routines?

Reply:
They most certainly are NOT dummy routines! Thanks, fixed the comment.